android/MobiledgeXSDKDemo/build.gradle

```diff
@@ -1,6 +1,6 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
-project.ext.matchingengineVersion = "2.1.0"

```

Comment:
Version name and repo directory name are temporary until final 2.1.1 is published to maven-releases.